samples/bluetooth/direct_test_mode/prj.conf

```diff
@@ -17,6 +17,7 @@ CONFIG_RTT_CONSOLE=y
 CONFIG_UART_CONSOLE=n
 
 # Use necessary peripherals
+CONFIG_NRFX_PPI=y
```

Comment:
Please note that nrf5340 network core doesn't have PPI it will cause warning when building for it

Reply:
Right, need, to modify the fix